src/Themes/DotNest.Core.SDK.Theme/Recipes/DotNest.Core.SDK.Setup.recipe.json

```diff
@@ -127,29 +125,49 @@
         "Lombiq.Hosting.Tenants.Admin.Login",
         "Lombiq.Hosting.Tenants.Admin.Login.SubTenant",
         "Lombiq.Hosting.Tenants.Management.ForbiddenTenantNames",
-        "Lombiq.Hosting.Tenants.Management.HideRecipesFromSetup",
+        "Lombiq.Hosting.MediaTheme",
         "Lombiq.Privacy",
         "Lombiq.Privacy.ConsentBanner",
         "Lombiq.Privacy.FormConsent",
         "Lombiq.Privacy.RegistrationConsent",
 
         // Themes
         "TheAdmin",
-        "TheTheme"
+        "Lombiq.BaseTheme",
+        "DotNest.Core.SDK.Theme"
       ]
     },
     {
       "name": "themes",
       "admin": "TheAdmin",
-      "site": "TheTheme"
+      "site": "DotNest.Core.SDK.Theme"
     },
     {
-      "name": "Settings",
-      "HomeRoute": {
-        "Action": "Index",
-        "Controller": "Home",
-        "Area": "TheTheme"
-      }
+      "name": "mediatheme",
+      "ClearMediaThemeFolder": false,
+      "BaseThemeId": "Lombiq.BaseTheme"
+    },
+    {
+      "name": "recipes",
+      "Values": [
+        {
+          "executionid": "DotNest.Core.SDK.Web",
+          "name": "Lombiq.BaseTheme.LayersAndZones"
+        }
+      ]
+    },
+    {
+      "name": "media",
+      "Files": [
+        {
+          "SourcePath": "_MediaTheme/wwwroot/example.png",
```

Comment:
There are no such files on the source path.

Reply:
Yeah I'll change it to Assets instead of wwwroot. It was kind of convenient to go with folder names that we are "used to" but wwwroot is actually ignored in most cases so it makes sense to go with other folder names.